optimade.rst

```diff
@@ -2587,8 +2592,9 @@ The Filter Language EBNF Grammar
 
     (* Comparison operator tokens: *)
 
-    Operator = ( '<', [ '=' ] | '>', [ '=' ] | [ '!' ], '=' ), [Spaces] ;
-    BooleanOperator = [ '!' ], '=' , [Spaces] ;
+    Operator = ( EqOperator | IneqOperator ) ;
+    EqOperator = [ '!' ], '=' , [Spaces] ;
```

Comment:
I personally find names `EqOperator` and `IneqOperator` sort of counter-intuitive. It does not immediately occur to me that `EqOperator` contains `!=` as well, and `IneqOperator` does not contain `!=`. However, I cannot propose anything better than `ExactOperator` instead of `EqOperator` and `CompareOperator` instead of `IneqOperator`.

This comment should not be treated as a blocking the PR. Renaming rules later on would probably not break the compatibility.

Reply:
Hmm... good point. However, equality and inequality are also strictly comparisons.

How about EqualityOperator and RelativeComparisonOperator ?

Reply:
> How about EqualityOperator and RelativeComparisonOperator ?

Yes, I like these over the previous ones.